src/content/docs/tutorials/counter-app/_multiple-functions.md

```diff
@@ -47,7 +47,7 @@ type Result<T> = std::result::Result<T, Err>;
 you can simply import `Result` from `anyhow`:
 
 ```rust
-use anyhow::Result;
+use color_eyre::eyre::Result;
```

Comment:
I'd generally shorten this to color_eyre::Result (it doesn't show up as a re-export in docs.rs as a doc hidden attribute on it, but it is recommended in various parts of the color_eyre docs oddly).

Not a huge deal, just one for consistency.